src/main/resources/story/shop_api/PostCustomers.story

```diff
@@ -0,0 +1,16 @@
+Story: POST /cusomers/ test
+
+Scenario: check response body
+When I send HTTP POST to the relative URL '/customers/' with content: '{
+  "firstname": "#{generate(Name.firstName)}",
+  "lastname": "#{generate(Name.lastName)}"
+}'
+Then the response code is equal to '201'
+When I save response body to the scenario variable 'body'
+Then a JSON element from '${body}' by the JSON path '$' is equal to '{
+  "firstname": "Freddy",
+  "lastname": "Meyers",
+  "customer_url": "/shop/customers/374"
```

Comment:
Can I somehow insert regexp into the line `customer_url": "/shop/customers/374` of expected json to check that actual json matches  `customer_url": "/shop/customers/\d+`

Reply:
You can use step 'When I save a JSON element from 'json' by JSON path 'jsonPath' to scopes variable 'variableName'' to save the value from line#13, then use any of those steps to validate that it matches:
Then the response body $comparisonRule '$content' (insert regex in $content)
Then a JSON element by the JSON path '$jsonPath' is equal to '$expectedData' (insert regex in $expectedData)